blog/content/post/2020-04-03-trenchboot-nlnet-lz-validation.md

```diff
@@ -22,6 +22,18 @@ categories:
 
 ---
 
+**UPDATE:** Since this blog post has been released, we have added more features
```

Comment:
We shouldn't say that we added more features. It was supposed to be working at the time of writing. Write, that we added missing verification for TPM1.2 and kernel measurements.

Reply:
@miczyg1 It is reworded now